mlflow/server/handlers.py

```diff
@@ -756,8 +1053,63 @@ def _get_artifact_repo(run):
 @catch_mlflow_exception
 @_disable_if_artifacts_only
 def _log_batch():
+
+    def _assert_metrics_correctly_constructed(metrics):
+        for m in metrics:
+            print("This is a metric: ", m)
+            ## Key
+            k = m["key"]
+            _assert_required(k)
+            _assert_string(k)
```

Comment:
We can remove type validations from these methods since ParseDict will handle type validation. `assert_required` is still helpful.

Reply:
Since we're only going to be checking that required fields are present, perhaps `_assert_metrics_fields_present` is a more descriptive name to use for this method, ditto for params